deeds-dapp-common/src/test/java/io/meeds/deeds/service/TrialServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package io.meeds.deeds.service;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import io.meeds.deeds.elasticsearch.model.TrialContactInformation;
+import io.meeds.deeds.storage.TrialRepository;
+
+@SpringBootTest(classes = {
+  TrialService.class,
+})
+class TrialServiceTest {
+
+  private static final String firstname    = "firstname";
+
+  private static final String lastname     = "lastname";
+
+  private static final String position     = "position";
+
+  private static final String organization = "organization";
+
+  private static final String email        = "email";
+
+  @Autowired
+  private TrialService        trialService;
+
+  @MockBean
+  private TrialRepository     trialRepository;
+    
+  @Test
+  void testSaveTrialWhenNotExists() throws Exception {
+    when(trialRepository.existsByEmail(email)).thenReturn(false);
+    trialService.saveTrial(firstname, lastname, position, organization, email);
+    when(trialRepository.existsByEmail(email)).thenReturn(true);
```

Comment:
What are we testing here ? you should use `verify(...)` and test when the email is duplicated that the trial isn't saved. Besides, the email duplication rule seems not relevant anymore as discussed in discord room. We may should delete this business rule and let creating multiple trials using the same email.